src/language/lang_sw.php

```diff
@@ -0,0 +1,669 @@
+  'Attacking Scoreboard' =>
+    'Ukurasa wa kufanyia mashambulizi',
+  'No attacking the competition infrastructure. If bugs or vulns are found, please alert the competition organizers immediately.' =>
+    'Hairihusiwi kuvamia miundo mbinu ya shindano. Kama kuna udhaifu au kasoro imeonekana, tafadhali mjulishe kiongozi au muongoza shindano mara moja.',
+  'Sabotage' =>
+    'Hujuma',
+  'Absolutely no sabotaging of other competing teams, or in any way hindering their independent progress.' =>
+    'Hairuhusiwi kabisa kuhujumu timu nyingine zilizo katika shindano, au kwa namna yoyote ili kujaribu kuzuia maendeleo yao binafsi katika shindano hili.',
+  'Bruteforcing' =>
+    'Kuingilia kwa nguvu',
+  'No brute forcing of challenge flag/ keys against the scoring site.' =>
+    'Hairuhusiwi kuingilia kwa nguvu flag/ keys kinyume na inavyo takiwa kwenye ukurasa wa alama.',
+  'Denial Of Service' =>
+    'KUzuia huduma',
+  'DoSing the CTF platform or any of the challenges is forbidden.' =>
+    'Kuzuia huduma za mfumo wa CTF au shindano lolote lile imezuiliwa kabisa.',
+  'Legal' =>
+    'Kisheria',
+  'Disclaimer' =>
+    'Onyo',
+  'By participating in the contest, you agree to release Facebook and its employees, and the hosting organization from any and all liability, claims or actions of any kind whatsoever for injuries, damages or losses to persons and property which may be sustained in connection with the contest. You acknowledge and agree that Facebook et al is not responsible for technical, hardware or software failures, or other errors or problems which may occur in connection with the contest.' =>
+    'Kwa kushiriki katika shindano hili, unakubali kuiachia huru Facebook na wafanyakazi wake, pamoja na taasisi inayo andaa shindano kutoka katika dhima, ya madai au matendo ya aina yoyote ile la kimajeraha, uharibifu, au hasara ya watu na mali, ambayo wanaweza kuhalalishwa kwa kuwa na muunganiko na shindano hili. Unatambua na kukubali kuwa Facebook, Taasisi inayoandaa shindano,pamoja na wenzake haihusiki na makosa ya kiufundi, ya kivifaa au ya kiprogramu, au makosa yoyote au matatizo ambayo yatotokea na kuwa na muingiliano wa  aina yoyote na shindano hili.',
+  'If you have any questions about what is or is not allowed, please ask an organizer.' =>
+    'Kama una maswali yoyote kuhusu kinacho ruhusiwa na kisicho ruhusiwa , tafadhali muulize muandaaji.',
+  'Have fun!' =>
+    'Kuwa na furaha!',
+  'Name' =>
+    'Jina',
+  'Email' =>
+    'Barua pepe',
+  'Token' =>
+    'Ishara(Token)',
+  'Team Registration' =>
+    'Usajili wa Timu',
+  'Team Name' =>
+    'Jina la Timu',
+  'Password' =>
+    'Neno Siri (Nywila)',
+  'Choose an Emblem' =>
+    'Chagua Nembo',
+  'Sign Up' =>
+    'Jiunge',
+  'Register to play Capture The Flag here. Once you have registered, you will be logged in.' =>
+    'Jisajili kucheza Capture the Flag hapa. Mara utakapo jisajili, utaingia ndani moja kwa moja.',
```

Comment:
Suggested Change: 'Jisajili kucheza Capture The Flag hapa. Mara utakapo jisajili, utaingia'